cluster_pack/skein/skein_config_builder.py

```diff
@@ -22,17 +21,25 @@ def build_with_func(
         package_path: Optional[str] = None,
         additional_files: Optional[List[str]] = None,
         tmp_dir: str = packaging._get_tmp_dir(),
-        log_level="INFO"
+        log_level="INFO",
+        process_logs: Callable[[str], Any] = None
 ) -> SkeinConfig:
     """Build the skein config from provided a function
 
     The function is serialized and shipped to the container
 
-    Returns
-    -------
-    SkeinConfig
+    :param func: the function to execute remotely
+    :param args: the function's arguments
+    :param package_path: the path on distributed storage where to find the application package
+                         (pex, conda zip)
+    :param additional_files: additional files to ship to the cluster
+    :param tmp_dir: a temp dir for local files
+    :param log_level: default remote log level
+    :param log_output_path: the path on distributed storage where to upload the container logs
```

Comment:
It doesn't match with the function definition, the last parameter in process_logs and it is not a path

Reply:
done